Traceback (most recent call last):
  File "/home/ubuntu/Web-Scrapping/app.py", line 1, in <module>
    import dash
ModuleNotFoundError: No module named 'dash'
Traceback (most recent call last):
  File "/home/ubuntu/Web-Scrapping/app.py", line 1, in <module>
    import dash
ModuleNotFoundError: No module named 'dash'
Traceback (most recent call last):
  File "/home/ubuntu/Web-Scrapping/app.py", line 84, in <module>
    app.run_server(host='0.0.0.0', port=8050, debug=True)
    ^^^^^^^^^^^^^^
  File "/home/ubuntu/Web-Scrapping/venv/lib/python3.12/site-packages/dash/_obsolete.py", line 22, in __getattr__
    raise err.exc(err.message)
dash.exceptions.ObsoleteAttributeException: app.run_server has been replaced by app.run
Traceback (most recent call last):
  File "/home/ubuntu/Web-Scrapping/app.py", line 84, in <module>
    app.run_server(host='0.0.0.0', port=8050, debug=True)
    ^^^^^^^^^^^^^^
  File "/home/ubuntu/Web-Scrapping/venv/lib/python3.12/site-packages/dash/_obsolete.py", line 22, in __getattr__
    raise err.exc(err.message)
dash.exceptions.ObsoleteAttributeException: app.run_server has been replaced by app.run
Dash is running on http://0.0.0.0:8050/

 * Serving Flask app 'app'
 * Debug mode: on
Dash is running on http://0.0.0.0:8050/

 * Serving Flask app 'app'
 * Debug mode: on
Dash is running on http://0.0.0.0:8050/

 * Serving Flask app 'app'
 * Debug mode: on
Dash is running on http://0.0.0.0:8050/

 * Serving Flask app 'app'
 * Debug mode: on
Dash is running on http://0.0.0.0:8050/

 * Serving Flask app 'app'
 * Debug mode: on
Traceback (most recent call last):
  File "/home/ubuntu/Web-Scrapping/app.py", line 8, in <module>
    app = dash.Dash(__name__)
          ^^^^
NameError: name 'dash' is not defined. Did you mean: 'hash'?
Traceback (most recent call last):
  File "/home/ubuntu/Web-Scrapping/app.py", line 8, in <module>
    app = dash.Dash(__name__)
          ^^^^
NameError: name 'dash' is not defined. Did you mean: 'hash'?
Traceback (most recent call last):
  File "/home/ubuntu/Web-Scrapping/app.py", line 8, in <module>
    app = dash.Dash(__name__)
          ^^^^
NameError: name 'dash' is not defined. Did you mean: 'hash'?
Traceback (most recent call last):
  File "/home/ubuntu/Web-Scrapping/app.py", line 8, in <module>
    app = dash.Dash(__name__)
          ^^^^
NameError: name 'dash' is not defined. Did you mean: 'Dash'?
Dash is running on http://0.0.0.0:8050/

 * Serving Flask app 'app'
 * Debug mode: on
Dash is running on http://0.0.0.0:8050/

 * Serving Flask app 'app'
 * Debug mode: on
